src/woo_publications/publications/tests/test_topic_api.py

```diff
@@ -42,18 +44,19 @@ def test_403_when_audit_headers_are_missing(self):
             self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
     def test_api_key_result_in_401_with_wrong_credentials(self):
-        publication = PublicationFactory.create()
+        topic = TopicFactory.create()
```

Comment:
this testcase will also need a `MEDIA_ROOT` override, or make the factory not generate an image in this instance :)